express-notes-api/server.ts

```diff
@@ -0,0 +1,139 @@
+import express from 'express';
+import { readFile, writeFile } from 'node:fs/promises';
+
+type Note = {
+  id: number;
+  note: string;
```

Comment:
This doesn't match what is in data.json. Apparently you aren't using this type, or you'd be getting errors!